src/app/components/home-loader/index.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
+import classnames from 'classnames';
+
+import UstwoLogoSequenceCutout from 'app/components/ustwo-logo-sequence-cutout';
+
+class HomeLoader extends Component {
```

Comment:
I'm going to need some explanation about the mechanics of this component. I don't understand why it receives a props (which I think has a timer) and inside you have another timer to flip `hide`.